cypress/e2e/tests/pages/explorer/cluster-project-members.spec.ts

```diff
@@ -0,0 +1,47 @@
+import UsersAndAuthPo from '~/cypress/e2e/po/pages/users-and-auth/users-and-auth.po';
+import ClusterProjectMembersPo from '~/cypress/e2e/po/pages/explorer/cluster-project-members.po';
+
+const runTimestamp = +new Date();
+const runPrefix = `e2e-test-${ runTimestamp }`;
+
+const username = `${ runPrefix }-cluster-proj-member`;
+const standardPassword = 'standard-password';
+
+describe('Cluster Project and Members', () => {
+  it('Members added to both Cluster Membership should not show "Loading..." next to their names', () => {
+    const usersAdmin = new UsersAndAuthPo('_', 'management.cattle.io.user');
+
+    // this will login as admin
+    cy.login();
+
+    // create a standard user
+    usersAdmin.goTo();
+    usersAdmin.listCreate();
+
+    usersAdmin.username().set(username);
+    usersAdmin.newPass().set(standardPassword);
+    usersAdmin.confirmNewPass().set(standardPassword);
+    usersAdmin.saveCreateForm().click();
+
+    // add user to Cluster membership
+    const clusterMembership = new ClusterProjectMembersPo('local', 'cluster-membership');
+
+    clusterMembership.goTo();
```

Comment:
we're trying to reduce the number of page loads e2e tests make, which is basically the goTo's.

you should be able to add a `navTo` function in ClusterProjectMembersPo that uses existing PO's to click burger menu --> side menu to reach the c & p members page

Reply:
done